website/docs/r/inserted_media.html.markdown

```diff
@@ -35,4 +37,5 @@ The following arguments are supported:
 * `catalog` - (Required) The name of the catalog where to find media file
 * `name` - (Required) Media file name in catalog which will be inserted to VM
 * `vapp_name` - (Required) - The name of vApp to find
-* `vm_name` - (Required) - The name of VM to be used to insert media file
\ No newline at end of file
+* `vm_name` - (Required) - The name of VM to be used to insert media file
+* `eject_force` - (Optional; *v2.1+*) Allows to pass answer to question when ejecting with not stopped VM. True means "Yes" as answer to question. Default is `true`
```

Comment:
Great to see the *v2.1+* flag.

Comments:
- For the unwary user it's not clear what this "question" is. Need to add a word or two about context.
- "when ejecting with not stopped VM" -> "when ejecting from a VM which is powered on"

Reply:
Added message example